tests/FSharpLint.Core.Tests/Framework/TestFuzzyHintMatcher.fs

```diff
@@ -6,11 +6,13 @@ open FSharpLint.Rules.Helper.Hints
 open FSharpLint.Framework
 open FSharpLint.Framework.AbstractSyntaxArray
 open FSharpLint.Framework.HintParser
-open FSharpLint.Framework.HintParser.MergeSyntaxTrees
+open FSharpLint.Framework.MergeSyntaxTrees
 open NUnit.Framework
 open FParsec
 open TestUtils
 
+// fsharplint:disable MaxLinesInValue MaxLinesInMember
```

Comment:
@webwarrior-ws two things:
- let's explain why
- let's add a matching fsharplint:enable at the end